fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -86,6 +86,7 @@ public DefaultValueDef(boolean isSet, Expr expr) {
     private final TypeDef typeDef;
     private AggregateType aggregateType;
     private boolean isKey;
+    private Boolean isAllowNullTristate;
```

Comment:
This name is a little strange.  need to comment why we need `isAllowNull `  and `isAllowNullTristate` both

Reply:
yes, changed, added